Makefile

```diff
@@ -47,7 +47,7 @@ dist: buildnformat
 test: generate
 ifndef SKIP_TESTS
 	@echo "--> Running go test"
-	@export PATH=$$PWD/build:$$PATH; go test $(TESTARGS) -p 1 ./...
+	@export PATH=$$PWD/build:$$PATH; go test $(TESTARGS) -p 1 ./... | grep -v vendor
```

Comment:
From my understanding it only exclude vendor from test output, not from actually testing the vendor directory. 
The way your are doing it will result on grep command returning always a 0 return code even if our tests are failing.

I think that the result of `go list ./...` would do the job.

Did I miss something

Reply:
Hum yes I missed it